seisbench/models/dkpn.py

```diff
@@ -0,0 +1,784 @@
+import json
+from typing import Any
+
+import numpy as np
+import torch
+import torch.nn as nn
+import torch.nn.functional as F
+from packaging import version
+
+import seisbench.util as sbu
+from seisbench.models.base import WaveformModel, _cache_migration_v0_v3
+
+# ---------  For PreProc class
+from obspy.core import Trace
+import copy
+from obspy.signal.filter import bandpass
+from scipy.signal import lfilter
+
+
+class DKPN(WaveformModel):
+    """
+    .. document_args:: seisbench.models PhaseNet
+    """
+
+    _annotate_args = WaveformModel._annotate_args.copy()
```

Comment:
`default_args` should be specified (and documented) here, so that they appear correctly in the documentation.